spec/support/shared/functional/windows_script.rb

```diff
@@ -52,6 +54,8 @@
   shared_examples_for "a script resource with architecture attribute" do
     context "with the given architecture attribute value" do
       let(:resource_architecture) { architecture }
+      # TODO: Why does expected_architecture default to :i386 and not
+      # the machine architecture?
```

Comment:
Wow the stubs in the test were not doing the right thing. It was always testing i386 resource architecture and guard architecture.